examples/shopping-cart/public/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+
+    <meta name="description" content="Web site created used create-prodo-app">
```

Comment:
```suggestion
    <meta name="description" content="Web site created using create-prodo-app">
```
(I see this is actually a typo in our template, so I'm also planning to fix that there: https://github.com/prodo-dev/prodo-template/pull/15 )